src/ensembl/src/shared/components/old-selected-species/index.ts

```diff
@@ -14,4 +14,5 @@
  * limitations under the License.
```

Comment:
What's the purpose of keeping the old-selected-species? No other component in our code seems to be using it.

Reply:
Missed it! That's now removed